File: messaging/v1/phone_number/create-service-multiple-number-add/create-service-multiple-number-add.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Messaging.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var phoneNumber = PhoneNumberResource.Create(
            phoneNumberSid: "PN2a0747eba6abf96b7e3c3ff0b4530f6e",
            pathServiceSid: "MG2172dd2db502e20dd981ef0d67850e1a"
        );

        Console.WriteLine(phoneNumber.Sid);
    }
}

File: messaging/v1/phone_number/create-service-multiple-number-add/create-service-multiple-number-add.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

phone_number = @client.messaging
                      .services('MG2172dd2db502e20dd981ef0d67850e1a')
                      .phone_numbers
                      .create(
                         phone_number_sid: 'PN2a0747eba6abf96b7e3c3ff0b4530f6e'
                       )

puts phone_number.sid
